docs/resources/examples/change_domain.yaml

```diff
@@ -7,10 +7,10 @@
 # Allowing FastDDS and ROS2 HelloWorld demo examples topics
 
 allowlist:
-  [
-    {name: "HelloWorldTopic", type: "HelloWorld"},                          # 1
-    {name: "rt/chatter", type: "std_msgs::msg::dds_::String_"},             # 2
-  ]
+  - name: "HelloWorldTopic"                                                 # 1
```

Comment:
In general:
Modify docs examples so they are links to the examples in resources. Aplpy the suggestions to the one file that remains (recommended the one in resources).
(This should have been like that from the beggining, I do not know at what time it changed)

Reply:
:+1: